lib/land/trackers/api_tracker.rb

```diff
@@ -206,24 +227,73 @@ def user_agent
                      Land.config.blank_user_agent_string
 
         @user_agent = UserAgent[user_agent]
+        @user_agent.user_agent_type = UserAgentType['user']
 
         if Land.config.identify_crawlers && defined?(CrawlerDetect)
           crawler_detect = CrawlerDetect.new(user_agent)
-          user_agent_type = crawler_detect.is_crawler? ? 'crawl' : 'api'
-          @user_agent.update(user_agent_type: UserAgentType[user_agent_type])
+          @user_agent.user_agent_type = UserAgentType['crawl'] if crawler_detect.is_crawler?
         end
 
         browser = ::Browser.new(user_agent)
+        land_browser = Land::Browser[browser.name]
+        update_browser_color_preferences(land_browser)
```

Comment:
Let's pull this function inline, and eliminate unnecessary checks:
```suggestion
        browser.dark_mode = dark_mode
        browser.light_mode = light_mode
        browser.no_preference = no_preference
```